src/middlewared/middlewared/pytest/unit/plugins/test_smart.py

```diff
@@ -38,45 +67,64 @@ def test__parse_smart_selftest_results__ataprint__1():
 
 
 def test__parse_smart_selftest_results__ataprint__2():
-    assert parse_smart_selftest_results(textwrap.dedent("""\
-        Num  Test_Description    Status                  Remaining  LifeTime(hours)  LBA_of_first_error
-        # 1  Offline             Self-test routine in progress 100%         0         -
-    """)) == [
+    data = json.loads("""\
```

Comment:
Why `json.loads`? This is a perfectly fine Python literal itself

Reply:
That's actually a really good point, I'll update those tests to just use a dict